### PR TITLE
Update with more citation data

### PR DESCRIPTION
**Pull request by michaelfaerber**

### CODE REVIEW

**Comment by napsternxg** [comment]
@michaelfaerber thanks. Can you provide details about the Pull Request?

**Comment by michaelfaerber** [comment]
I'll to contribute to the awesome list of data sets by adding the data sets (1) Microsoft Academic Knowledge Graph (http://ma-graph.org/), (2) arXiv CS dataset (http://citation-recommendation.org/publications), and (3) unarXiv (https://github.com/IllDepence/unarXive).

File: README.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ List of resources on scholarly data analysis ranging from datasets, papers, and
 * [OSF API (Open Science Framework)](https://developer.osf.io/)
 * [Digital tools for researchers](http://connectedresearchers.com)
 * [Fatcat - versioned, publicly-editable catalog of research publications](https://fatcat.wiki/)
+* [Microsoft Academic Knowledge Graph](http://ma-graph.org/)
```
Comment (napsternxg) [inline README.md]: Can you change the name of Microsoft Academic Knowledge Graph to Microsoft Academic Graph - RDF dump as we already have MAG at the top. Another option is to add a separate link next to MAG with title RDF triples.

File: README.md [hunk 2]
```diff
@@ -78,6 +78,9 @@ List of resources on scholarly data analysis ranging from datasets, papers, and
...
+* [arXiv CS dataset](http://citation-recommendation.org/publications/#A_High-Quality_Gold_Standard_for_Citation-based_Tasks)
```
Comment (napsternxg) [inline README.md]: For arXiv CS dataset, rename it to arXiv CS citation in context, may be put it under `Information Extraction and NLP`

File: README.md [hunk 3]
```diff
@@ -78,6 +78,9 @@ List of resources on scholarly data analysis ranging from datasets, papers, and
...
+* [arXiv fulltext + citations dataset](https://github.com/IllDepence/unarXive)
```
Comment (napsternxg) [inline README.md]: Keep this but change the link to the zenodo page: https://zenodo.org/record/2609187#.XKe86JhKh3g

For the code also add an entry for the `unarXiv (Citation in context from arXiv)` put it under `Tools > Citation and metadata extraction`. 
Also, keep the original but link to the zenodo page:

**Review by napsternxg — CHANGES_REQUESTED** [review]
@michaelfaerber thanks a lot, these are all very exciting resources. 
I have made some comments about arranging the links. Can you do that?

**Comment by napsternxg** [comment]
I have resolved the changes and am merging these. 
Thanks for the links @michaelfaerber

**Pull request merged**